_includes/docs/user-guide/ui/unit-conversion.md

```diff
@@ -0,0 +1,114 @@
+* TOC
+{:toc}
+
+{% assign sinceVersion = "4.1" %}
+{% include templates/since.md %}
+
+Starting from ThingsBoard 4.1, widgets support automatic data conversion and unit switching based on the selected unit system. 
+This is especially useful in global IoT solutions and significantly improves data interaction for users across different geographic regions where various measurement standards are used — such as the metric system (°C, km, kg) and the imperial system (°F, miles, lbs).
+
+<br><b><font size="4">Key benefits:</font></b>
+
+- Automatic conversion of numerical values into the appropriate units directly within the widget — no additional data processing required.
+- Flexible widget configuration that eliminates the need to duplicate widgets or create separate dashboards for users with different regional unit standards.
+- Support for a wide range of measurement units, allowing users to view values in the units that are most familiar and convenient to them.
+
+<b><font size="4">Where the unit conversion feature can be applied?</font></b>
+
+All widgets designed to display telemetry data include a unit conversion option.   
+Unit conversion is supported for telemetry data, Y-axis scales, and threshold settings.
+
+> **Note:** If a widget contains multiple telemetry keys, the unit conversion feature must be configured individually for each key.
+
+## Enable unit conversion
+
+Unit conversion for telemetry keys, the Y-axis, and threshold values is configured in the widget&#39;s "**Unit settings**" section.
+- Specify the **source unit** – This tells the system which unit the current value is measured in, so it knows how to convert it.
+- Enable unit conversion by toggling the "**Enable unit conversion**" switch.
+- The system will automatically suggest commonly used target units for each unit system (e.g., °C for Metric, °F for Imperial, and °C for Hybrid). However, you can override these by selecting different units from the dropdown list.
+
+> **Note**: All available units are grouped by category (e.g., temperature, length, mass). You can only select target units from the same category as the source unit.
+
+> **Note**: If you do not specify a unit for a particular unit system, that system will fall back to using the unit defined in the **Source unit** field.
+
+- Click "**Apply**" to save the changes.
+
+Once unit conversion is enabled, an icon will appear in the Unit field indicating that the feature is active.
+
+{% include images-gallery.html imageCollection="unit-conversion-configuring" %}
+
+### Range color settings
+
+**Range color settings** are based on the **original telemetry value**, before any unit conversion is applied.
```

Comment:
Here are a few options for clarity, incorporating the previous suggestion about source units:

Range color settings are based on the original telemetry value received from the back-end, in its source units, before any unit conversion is applied.
The range color settings use the original telemetry value as it arrives from the back-end (i.e., in its source units), prior to any unit conversion.
To ensure accurate range color settings, they are applied to the original telemetry value in its source units, before any unit conversions take place.